eox_tenant/models.py

```diff
@@ -65,6 +66,99 @@ def get_microsite_for_domain(cls, domain):
         return microsites[0] if microsites else None
 
 
+class TenantConfigManager(models.Manager):
+    """
+    Custom managaer for Tenant Config model.
+    """
+    def get_configurations(self, domain):
+        """
+        Execute optimized query to get site configurations.
+        """
+        from django.db import connection
+
+        with connection.cursor() as cursor:
+            cursor.execute("""
+                SELECT eox_c.id, eox_c.external_key, eox_c.lms_configs, eox_c.studio_configs, eox_c.theming_configs, eox_c.meta
```

Comment:
Could you break each item in a new line? Just to make the query more readable. @diegomillan